src/core/default/mappers/simple/lambda_deployer.py

```diff
@@ -496,7 +496,7 @@ def update_progress_bar(x):
 ##### Main Entry Point
 #######################
 
-
+# ANIBAL return type does not match
```

Comment:
Return should be Optional[Dict] because the _create and _update functions return a Dict of the Cloud Output, but the delete function does not return anything. This should be the return type of any Mapper function.